src/Server/HTTPHandler.cpp

```diff
@@ -884,6 +884,9 @@ void HTTPHandler::processQuery(
     {
         if (settings.http_write_exception_in_output_format && output_format.supportsWritingException())
         {
+            ExecutionStatus status = ExecutionStatus::fromCurrentException("", false);
```

Comment:
AFAIU, for second parameter shouldn't be just false but result of such check:
```
params.getParsed<bool>("stacktrace", false) && server.config().getBool("enable_http_stacktrace", true)
```

Reply:
I agree. Applying feedback.